db/seed_helper.rb

```diff
@@ -5,4 +5,15 @@ def create_roles_and_permissions
   Role.create!(:name => Role::DATA_OWNER_ROLE)
 end
 
-
+# Populates the languages table with the language names and codes from the languages CSV file
+def populate_languages
+  Language.delete_all
```

Comment:
Yes, for now this is true, but what if somebody decides to create associations to a Language from other models? You would save hours of debugging an issue that would show in production by fixing this now - it's a 1 line change.